src/toasts/toasts.stories.js

```diff
@@ -0,0 +1,21 @@
+import { storiesOf } from "@storybook/html";
+import html from "./toasts.stories.html";
+import "./toasts.css";
+import ejs from "ejs";
+
+import { htmlStory } from "../story-helpers";
+import { showToast } from "./toasts.js";
+// storiesOf("OpenMRS Styleguide", module).add(
```

Comment:
Did you intend to leave these in?

Reply:
No.

Reply:
I need to finish this file. I have to create a story for this toast. This comment is temporary.